recipe/meta.yaml

```diff
@@ -10,33 +10,43 @@ source:
   sha256: eebca7f587b1761ce43a98ffd5327f2b600b9aa5bb0a77b947687f503ad43bc0
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - cryptography
+    - pycryptodome
```

Comment:
`pycryptodome` is an optional dependency, see https://github.com/authlib/joserfc/blob/2470d72dec9aa15c316325aafc35ca41ec638f66/pyproject.toml#L30-L31.
Only `cryptography` is required https://github.com/authlib/joserfc/blob/b669af5fbf17ad500893004299debe006885dd97/pyproject.toml#L5-L7

It's not mentioned in the source code but docs https://github.com/search?q=repo%3Aauthlib%2Fjoserfc%20pycryptodome&type=code